frontend/src/common/queries/differentialExpression.ts

```diff
@@ -243,31 +245,52 @@ export function useAvailableOrganisms() {
 export function useDifferentialExpression(): {
   data: DifferentialExpressionQueryResult;
   isLoading: boolean;
+  errorMessage: string | null;
 } {
   const requestBody = useDEQueryRequestBody();
   const { data, isLoading } = useDEQuery(requestBody);
   return useMemo(() => {
-    if (isLoading || !data)
+    if (isLoading || !data || data.successCode === 1) {
+      let errorMessage;
+      switch (true) {
+        case isLoading:
+          errorMessage = "Loading results";
```

Comment:
nit: semantically "loading results" is not an error message, so maybe just `message` instead `errorMessage` for the key name?